docs/ui/emacs.rst

```diff
@@ -159,9 +161,12 @@ is open.
 
 You can toggle the interactive mode with ``C-c C-i``. If you were to do
 so, you still can navigate the proofs with ``C-c C-p`` and ``C-c C-n``,
-and show the goals at the current line with ``C-C C-c``. As always with
-emacs, if you were to be dissatisfied with these keybindings, you can
-change them easily!
+or ``C-c C-b`` and ``C-c C-f``, and show the goals at the current line 
```

Comment:
What are C-c C-b and C-c C-f for?

Reply:
Jumping to the previous and next proof. Should I add that to the docs?

Reply:
Yes, please. Thank you.